frontend/components/organisms/Comment/index.tsx

```diff
@@ -29,7 +38,7 @@ const Comment = ({ id, text, author, time, action, userId, slug, refetchHandler
         <div className="group flex flex-wrap gap-2 px-2 py-4">
             <span className="mr-4">{text}</span>
             <span>
-                <Link href={`/users/${slug}`} className="text-blue-500 hover:text-blue-400">
+                <Link href={`/users/${slug ?? ''}`} className="text-blue-500 hover:text-blue-400">
```

Comment:
I don't think the slug will ever be null?

Reply:
The slug in the prop definition is marked as optional that's why ESLint will complain that the slug might be null